### ISSUE TITLE
Attribute naming error: name != title

### ISSUE THREAD

**Issue by mrjj** [w3c/pub-manifest#202]
Between EPUB and pub/wpub/dpub worlds there is blinking attribute (property) `title`<->`name`. 

From any possible archivist and bibliographic point of view its very rough error:
It is as bad as comment that name is actually a title under hash anchor `pub-title`:

```
name | Human-readable title of the publication. | One or more Text. | Array of Localizable Strings | name (Thing)
```
https://w3c.github.io/pub-manifest/#pub-title

Yes, there is such very uncommon thing like `publication name` like "Gutenberg Bible" (and its actually title-less but containing bible text) and this is not `publication title`. As well as its no way the same as the person's `name` and `title` - its really bad case of cross-polysemy. And anyway in bibliography "Gutenberg Bible" will be the `access title`/`index title`/`alternate title` just not to play on border. [Here](https://english.stackexchange.com/questions/173675/whats-the-difference-between-title-and-name) you can see very professional explanation of the details.

For this kind of complex cases IMO rule of thumb is to use term that domain is using, for completely all kinds of publication, because usually domain resolved this kind of naming problems long time ago. E.g. i really happy to see that `digital` finally overcomed `electronic` because last one war just term error, (you can print pub bytecode on paper and it will stay digital and conforming data/metadata standard, but not electronic any possible way) So IT domain corrected publishers error in its domain field right.

In bibliographic and publishers world that is sometimes very tangled in terms the solid domain pillar [DCMI](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) is 0-conflict worldwide accepted authority core terms set. No other standards are expecting to be in conflicts with it.

I understand the initial reason of `name` field property is aligning with core ontology of schema.org including all-in-one mega-class of [CreativeWork](https://schema.org/CreativeWork) and one standard is aligning with linked data and other one with bibliographic heritage. Now this mega-class is curated by some group but it happened after its "initialization" with name property among lot of other major classes. They actually have a lot of errors like `alternativeHeadline` that commented as an `A secondary title of the CreativeWork` all wrong, headline its what is printed as second line of periodic edition, title may be `The Sun` and headline: `New cannibal horse assault on bishop!`

As well as about their `alternateName` field for alternate variant of very-very uncommon thing that defined by schema.org :)

TL;DR **name ! = title** and its not an any kind of synonyms being related and to publications and to persons. 

I expecting web standards to avoid so dramatic domain errors and legacy and compatibility reasons could be the reason to stay with errors of this kind.

schema.org seems to be the source of this error, and should fix it as well. I'll write the similar problem report to their [CreativeWork](https://schema.org/CreativeWork) maintainers group.

**Comment by iherman** [comment]
@mrjj I believe nobody is happy with the usage of the term `name`. However, one of the starting positions of the manifest specification was to align the terms, as much as possible, to the schema.org terminology. Alas!, they use the term `name`, hence this choice.

**Comment by mrjj** [comment]
Thank you for your answer. 
I realising how significant the change i'm talking about. I've heard "nobody is happy" this is exactly what i want. 
So, well, its good news that Schema.org have a field to define a publication's name. As well that WPM is aligned in fields set. So my proposal seems to be only about adding the missing and very important field title as the key matter of proposal. On technical level extension of the fields set is not breaking change. And OCLC as the introduction point for this extension proposal and "fix delivery". Major founding institutions may stand as the reserve introduction points.
I'm pretty sure that the WPM working group have more important things to do, thank you for your time and your work!
